src/components/molecules/EmergencyAlertCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { StyleSheet } from 'react-native';
+import { View, Text, Divider, VStack, HStack, Pressable } from 'native-base';
+import moment from 'moment';
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { RootStackParamList } from '../../navigation/types';
+import { Geolocation } from '../../utils/geolocation';
+
+type EmergencyAlertCardProps = {
+  sender: string;
+  time: Date;
+  title: string;
+  description: string | undefined;
+};
+
+export interface EmergencyData {
+  elderlyImageId: string;
+  elderlyName: string;
+  location: Geolocation;
+  timestamp: Date;
+  elderlyPhone: string;
+}
+
+export const mockEmergencyInfo = {
+  name: 'That dude',
+  address:
+    'some random location too long to describe in a single screen of Aegis mobile application',
+  location: {
+    latitude: 13.0,
+    longtitude: 25.0
+  },
+  date: moment(new Date(2022, 8, 12)).format('DD MMM YYYY'),
+  time: moment(new Date(2022, 8, 12)).format('hh:mm:ss'),
+  phone: '0855555555',
+  elderlyImageId: 'abcdef'
+};
```

Comment:
Just curious. Is the emergency information working for normal cases now or is it only mock info for now?

Reply:
This is the placeholder location used before the actual service is done. So the service is working now. Except for reverse geocoding of course @pointmekin

Reply:
Noted.